### DISCUSSION TITLE
Value of type 'Transaction' has no member 'environment'

### DISCUSSION THREAD

**Discussion by rudraiGeek** [hyochan/react-native-iap#2394] [Q&A]
I am trying to run the project which has RNIap installed, but I am getting this error in a swift file named `IapSerializationUtils.swift`.
I am attaching the screenshot, please check.
I am on MacOs Monterey 12.2.1 using Xcode 13.2.1

![IAP](https://user-images.githubusercontent.com/120083725/229111600-084703c7-ac31-445a-b0d0-e2308e7c1101.png)

**Comment by spirift** [comment]
I had the same issue, it drove me mad for days. I was using bitrise and upgrading xcode to 14.3.x and macOS 13.2 (Ventura) solved this issue for me. Hope it helps.
